definitions/contested/json/AuthorisationCaseField/ExpressPilot-nonprod.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "LiveFrom": "21/02/2025",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "suitableForExpressCaseLabel",
+    "UserRole": "caseworker-divorce-financialremedy-solicitor",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "21/02/2025",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "suitableForExpressCaseLabel",
+    "UserRole": "caseworker-divorce-financialremedy-courtadmin",
+    "CRUD": "CRUD"
```

Comment:
I am not sure if "R" is enough.

Reply:
Hi @so99y.  I know what you mean.  I tried R, CR, then when that didn't work I used CRUD as that is what the recent Draft Order labels used. 

From an access perspective, this is label, so no concerns from a data access perspective.  I'll raise.